pkg/kubelet/kubelet.go

```diff
@@ -3404,7 +3404,11 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return err
 	}
 	// Update the current status on the API server
-	_, err = kl.kubeClient.Core().Nodes().UpdateStatus(node)
+	updatedNode, err := kl.kubeClient.Core().Nodes().UpdateStatus(node)
+	if err == nil {
+		kl.volumeManager.MarkVolumesAsReportedInUse(
```

Comment:
as a follow up, we can consider a force sync. Something like:
``` 
    select {
     case <- forceSync
     case <-time.After(10*time.Second)
     }
    tryUpdateNodestatus()
```

And then pass the forceSync channel down to the volume manager. Something to consider at least, to cut out the 10s delay.